Lib/idlelib/README.txt

```diff
@@ -259,3 +259,21 @@ htest function def or "if __name__ == '__main__'" clause.
 Within module imports like "from idlelib.mod import class" may cause
 circular imports to deadlock.  Even without this, circular imports may
 require at least one of the imports to be delayed until a function call.
+
+What's New entries
+------------------
+
+Repository directory Doc/whatsnew/ has a file 3.n.rst for each 3.n
+Python version.  For the first entry in each file, add subsection
+'idlelib and IDLE', in alphabetical position, to the 'Improved Modules'
+section.  The first entry to '3.(n+1).rst is the first feature or
```

Comment:
Either drop the single-quote, or add a closing single-quote.

Reply:
Also, I find that sentence hard to comprehend.

Reply:
Quote dropped.  Is the expanded rewrite clearer?

Reply:
Thanks, it is!

Reply:
Thank you for giving me a possible target audience view.